### ISSUE TITLE
Lots of links are not working?

### ISSUE THREAD

**Issue by yxchng** [google-research-datasets/conceptual-12m#4]
```
16it [21:34, 19.91s/it]worker  - success: 0.244 - failed to download: 0.752 - failed to resize: 0.003 - images per sec: 8 - count: 10000
total   - success: 0.241 - failed to download: 0.755 - failed to resize: 0.003 - images per sec: 124 - count: 160000
```

**Comment by joliver1981** [comment]
Yeah, the vast majority of the links don't appear to work anymore. I think you can download as tfrecords from kaggle. Search for "cc12m" on kaggle datasets. Wonder if anyone knows if there is an updated weights file for download that has been trained on these images.

**Comment by yxchng** [comment]
@joliver1981 Have you tried downloading the Kaggle's cc12m? How many images are there?

**Comment by Suhail-BW** [comment]
Due to vast majority of links not working here, I suggest HuggingFace Datasets which offers older version of cc12m which had fewer broken links. Consider using it as an alternative. https://huggingface.co/datasets/pixparse/cc12m-wds
